README.md

```diff
@@ -1,11 +1,108 @@
-# Create .env files in order to run locally
+# NC News API
+
+## https://gareths-nc-news.herokuapp.com/api/
+
+<hr>
+
+## Background
+
+I have built an API for the purpose of accessing application data programmatically. The intention is to mimic a real world backend service which should provide information to frontend architecture.
```

Comment:
Don't talk in the first person. Talk about the API. This API is....

Also, it isn't mimicing a back-end service, it is a real back-end service! :)

Reply:
Is this better?

Reply:
Sure thing - thank you